sample-java-model/src/main/java/com/vimeo/sample_java_model/SwappableParserExampleModel.java

```diff
@@ -0,0 +1,28 @@
+package com.vimeo.sample_java_model;
+
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A model to test the case where the consumer
+ * might try to use different parsers for
+ * an object across different gson instances.
+ */
+@UseStag
+public class SwappableParserExampleModel {
```

Comment:
Actually, I have to take that back. They have to be public because the generated Stag.Factory that references both this class and the inner class is being generated in `com.vimeo.sample_java_model.stag.generated` which is outside the package.